domains/zishu.localplayer.dev.json

```diff
@@ -0,0 +1,13 @@
+{
+    "description": "Will be used on my local server that I working on.",
+    "domain": "localplayer.dev",
+    "subdomain": "zishu",
+    "owner": {
+        "repo": "https://github.com/HP-L/hp-l.github.io",
+        "email": "lake.hupo.2001@outlook.com"
+    },
+    "record": {
+        "NS": "cullen.ns.cloudflare.com"
```

Comment:
NS records should be in an array. Also NS records are no longer supported unless donated per readme.